presto-docs/src/main/sphinx/presto_cpp/properties.rst

```diff
@@ -323,3 +323,37 @@ This only applies if ``system-mem-pushback-enabled`` is ``true``.
 
 Specifies the amount of memory to shrink when the memory pushback is
 triggered. This only applies if ``system-mem-pushback-enabled`` is ``true``.
+
+Environment Variables As Values For Worker Properties
+-----------------------------------------------------
+
+This section applies to worker configurations in the config.properies file
+and catalog property files only.
+
+The value in a key-value par may reference an environment variable by using
+a leading `$` followed by enclosing the environment variable name in brackets (`{}`).
+
+```
+key=${ENV_VAR_NAME}
+```
+
+The environment variable name must match exactly with the defined variable.
+
+This allows a worker to read sensitive data e.g. for access keys from an
+environment variable rather than having the actual value hard coded in a configuration
+file on disk enhancing the security of deployments.
```

Comment:
```suggestion
file on disk, improving the security of deployments.
```
Comma suggestion to separate the end of the sentence from the main idea for clarity. Very minor nit suggestion of "improving" as a more common and understandable word for better readability, but please revert that to "enhancing" if you feel this nit suggestion doesn't accurately describe your meaning.